HWs/HW2/Fibonacci.c

```diff
@@ -0,0 +1,54 @@
+#include <stdio.h>
+#include <stdbool.h>
+
+#define bound 47
+
+int iteratFibonacci(int number)
```

Comment:
```suggestion
int iterativeFibonacci(int number)
```
Такие сокращения не приветствуются